### DISCUSSION TITLE
Airflow Scheduling using cron or Airflow preset

### DISCUSSION THREAD

**Discussion by MR-GOYAL** [elyra-ai/elyra#3212] [General]
Hi team,

Is there any roadmap on adding scheduling of Airflow dags on cron or Airflow preset? Currently, scheduling is hardcoded to once only.

**Comment by lresende** [comment]
Good timing, we were discussing this yesterday on the community call and here is a related issue -> https://github.com/elyra-ai/elyra/issues/471 ... Having said that, I believe @D3f0 was mentioning kfp as first priority, so if you want to collaborate on the Airflow side, that would be good.

**Reply by MR-GOYAL** [reply]
Actually i did some changes to support cron and presets and tested with Airflow 2.6.2 and its working as expected ( just writing unit test cases are pending). However, I just found out that Airflow 2.0 support is still in progress. How  to proceed in this case?

**Comment by lresende** [comment]
I believe the Airflow 2.0 that is in progress is mostly around custom components. I would say, provide a pull request and we will deal with merges as needed.

**Reply by MR-GOYAL** [reply]
I am struggling to write unit test cases. Might need help on that.
